### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by sahehb**

### CODE REVIEW

File: mevislab.github.io/content/tutorials/testing/testingexample1.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ menu:
 
 {{< youtube "DqpVaKai_00" >}}
 
+{{< youtube "DqpVaKai_00" >}}
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/testing/testingexample1.md:18]: Why is the link needed twice here?

File: mevislab.github.io/content/tutorials/basicmechanisms/macromodules/guidesign.md [hunk 2]
```diff
@@ -305,10 +305,36 @@ def myButtonAction:
...
+* {{< docuLinks "/Resources/Documentation/Publish/SDK/MDLReference/index.html#TestEventFilter" "TestEventFilter Module">}}
+* {{< docuLinks "/Resources/Documentation/Publish/SDK/MDLReference/index.html#TestStyles" "TestStyles Module">}}
+* {{< docuLinks "/Standard/Documentation/Publish/ModuleReference/TestButtonGroups.html" "TestButtonGroups Module">}}
+* {{< docuLinks "/Standard/Documentation/Publish/ModuleReference/TestImageMap.html" "TestImageMap Module">}}
+
+Please use the *Module Search* with the prefix *Test* for more examples.
+
 ## Summary
 * User interfaces and several module panels can be created for each macro module.
 * You can create a panel, define in and outputs as well as interactions, in your *.script* file in {{< docuLinks "/Resources/Documentation/Publish/SDK/MeVisLabManual/ch26.html" "MATE">}} by using the {{< docuLinks "/Resources/Documentation/Publish/SDK/MDLReference/index.html" "MeVisLab Definition Language (MDL)">}}.
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/basicmechanisms/macromodules/guidesign.md:336]: inputs and outputs

File: mevislab.github.io/content/tutorials/testing/testingexample1.md (hunk 1 above)
Comment (sahehb) [reply mevislab.github.io/content/tutorials/testing/testingexample1.md:18]: Merge error

Comment (mrbean-bremen) [reply mevislab.github.io/content/tutorials/testing/testingexample1.md:18]: Will you correct it?

Comment (sahehb) [reply mevislab.github.io/content/tutorials/testing/testingexample1.md]: i did

File: mevislab.github.io/content/tutorials/dataobjects/curves/curvesexample1.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+10 10 20
+50 50 100
+```
+{{</highlight>}}
+
+You can see 2 curves. The second and third columns are printed as separate curves. Both appear yellow. After checking *Split columns into data sets*, you will see one yellow and one red curve.
+
+{{<imagegallery 2 "/images/tutorials/dataobjects/curves" "before_split" "after_split">}}
+
+If the flag *Split columns into data sets* is set to *TRUE*, then a table with more than two columns is split into different *CurveData* objects. This gives the user the possibility to assign for each series a different style and title.
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/dataobjects/curves/curvesexample1.md:76]: ... to assign a different style and title for each series.

**Pull request merged**